Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1

```diff
@@ -21,6 +21,17 @@ Write-Verbose -Verbose "configFile = $configFile"
 Write-Verbose -Verbose "dbConnectionString = $dbUrl"
 Write-Verbose -Verbose "dbUsername = $dbUsername"
 
+$prcaEnabled = Get-TaskVariable -Context $distributedTaskContext -Name "GitPullRequestCodeAnalysisEnabled"
```

Comment:
Mathew was insisting on using CodeAnalysis instead of SonarQube.. but agree that CodeAnalysis is too generic a term. I'll have a chat with Mathew and change this..